peachjam/js/components/FindDocuments/AdvancedSearch.vue

```diff
@@ -0,0 +1,223 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <form @submit.prevent="$emit('submit')">
+        <div class="row">
+          <div class="col-12 mb-3">
+            <label for="global">{{ $t("Search all documents:") }}</label>
+            <input
+              id="global"
+              name="global"
+              type="text"
+              class="form-control"
+              :value="globalSearchValue"
+              :aria-describedby="$t('Search all documents:')"
+              :placeholder="$t('Search documents by all fields')"
+              @input="onGlobalSearch"
+            >
+          </div>
+          <div class="col-6 mb-3">
+            <div class="form-group">
+              <label for="title">{{ $t("Title:") }}</label>
```

Comment:
I suggest moving the colon outside of the translation string, so that we don't have to translate both "Title:" and "Title".